Classes/ScrnAA12AutoShotgun.uc

```diff
@@ -85,7 +85,7 @@ simulated function ClientReload()
     if ( MagAmmoRemaining <= 0 || ReloadShortAnim == '' )
     {
         PlayAnim(ReloadAnim, ReloadAnimRate*ReloadMulti, 0.1);
-        FakedShell.Destroy();
+        FakedShell.Destroy(); //try destroying the faked shell to prevent it from appearing
```

Comment:
Shouldn't we check for `if ( FakedShell != none)` ?

Reply:
Oh yeah, I should have added that.
There is currently still a bug where if you do exactly one tactical reload after buying the weapon and then do an empty reload afterwards, the fakedshell still appears. Maybe it has something to do with how the values are initialized, or maybe it's related to this, I need to test it more.